templates/accounts/resend_verification_email.html

```diff
@@ -0,0 +1,30 @@
+{% extends 'layout.html' %}
+
+{% load static %}
+
+{% block title %} Jagrati | Resend activation mail {% endblock %}
+
+
+{% block content %}
+
+<div class="container pt-3" style="margin-top: 100px;">
+    <div class="text-center mt-4 p-0 p-sm-3" style="margin: auto; max-width: 500px;">
+        <img src="{% static 'accounts/icon/mail-sent.gif' %}" alt="Email Sent" style="max-width: 200px;">
+        <h1 class="mt-2 font-weight-bolder">Resend Activation mail</h1>
```

Comment:
Can you add some more margin around this heading?

Reply:
I didn't understand.

Reply:
Some margin below the `h1` tag (to give some gap between the heading and the textbox).

Reply:
@garg3133

Reply:
![image](https://user-images.githubusercontent.com/61455329/112680592-940abb00-8e93-11eb-9d40-dfc38ae28421.png)
Okay now?